src/components/Details.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import useDetails from '../hooks/use-details';
+import Header from './Header';
+import '../style/common.css';
+import './Details.css';
+
+
+
+const Details = () => {
+  const details = useDetails();
+  return (
+    <>
+      <Header subhead={'Character Details'}/>
+      <main>
+        <div>
+          <img src={details.photoUrl} />
+          <dl>
+            <dt>Name:</dt>
+            <dd>{details.name}</dd>
+            <dt>Position:</dt>
+            <dd>{details.position}</dd>
+            <dt>First Episode:</dt>
+            <dd>{details.first}</dd>
```

Comment:
I'm not sure if it's the api, but many of the characters' detail page had `First Episode` as `"`, with no closing quotation mark. You might consider adding a fallback if the api doesn't return a first episode (I checked characters like Aang, to make sure I wasn't just clicking on an obscure character from the show)